### PR TITLE
Moved Custom Commands to H2 Database

### PR DESCRIPTION
**Pull request by DynxstyGIT**

### CODE REVIEW

File: src/main/java/net/javadiscord/javabot/command/SlashCommands.java [hunk 1]
```diff
@@ -205,45 +206,44 @@ private CommandConfig findCommandConfig(String name, CommandConfig[] configs) {
...
+			} else {
+				if (reply) return event.reply(responseText);
+				else {
+					return RestAction.allOf(event.getChannel().sendMessage(responseText), event.reply("Done!").setEphemeral(true));
+				}
+
 			}
+		} catch (SQLException e) {
+			e.printStackTrace();
+			return null;
```
Comment (MoonTM-GIT) [inline src/main/java/net/javadiscord/javabot/command/SlashCommands.java:244]: Why not send a message saying something went wrong? (e.g. `Responses.error`)

File: src/main/java/net/javadiscord/javabot/systems/staff/custom_commands/dao/CustomCommandRepository.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+			rs.close();
+		}
+		return Optional.ofNullable(command);
+	}
+
+	/**
+	 * Gets all custom commands for the given guild
+	 * @param guildId The id of the guild.
+	 * @return A List with all custom commands.
+	 * @throws SQLException If an error occurs.
```
Comment (MoonTM-GIT) [inline src/main/java/net/javadiscord/javabot/systems/staff/custom_commands/dao/CustomCommandRepository.java:128]: this doesn't throw anything

File: src/main/java/net/javadiscord/javabot/systems/staff/custom_commands/subcommands/CreateSubCommand.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+		if (nameOption == null || responseOption == null) {
+			return Responses.error(event, "Missing required arguments.");
+		}
+		var replyOption = event.getOption("reply");
+		boolean reply = replyOption == null || replyOption.getAsBoolean();
+
+		var embedOption = event.getOption("embed");
+		boolean embed = embedOption == null || embedOption.getAsBoolean();
+		System.out.println(reply);
+		System.out.println(embed);
```
Comment (MoonTM-GIT) [inline src/main/java/net/javadiscord/javabot/systems/staff/custom_commands/subcommands/CreateSubCommand.java:35]: remove these

**Review by MoonTM-GIT — CHANGES_REQUESTED** [review]

File: src/main/java/net/javadiscord/javabot/systems/staff/custom_commands/subcommands/CreateSubCommand.java (hunk 3 above)
Comment (DynxstyGIT) [reply src/main/java/net/javadiscord/javabot/systems/staff/custom_commands/subcommands/CreateSubCommand.java:35]: oh right lmao

**Review by MoonTM-GIT — APPROVED** [review]

**Pull request merged**